app/utils/constants.js

```diff
@@ -95,6 +95,7 @@ var C = {
     I_HATE_SPINNERS: 'iHateSpinners',
     EXPANDED_STACKS: 'expandedStacks',
     SORT_STACKS_BY: 'sortStacksBy',
+    DEFAULT_UI_THEME: 'defaultUiTheme'
```

Comment:
Can this just be called `THEME` / `theme` (and all the other places that say `defaultUiTheme`)?  All prefs are for the UI and this stores the one they picked, not the default.